.circleci/config.yml

```
@@ -81,6 +81,29 @@ commands:
           event: success
           condition: on_success
 
+  run_test_byte_code:
+    description: "Run test byte code on a environment."
```

Let's call this command `prepare_bytecode_report`.
```suggestion
  prepare_bytecode_report:
    description: "Generate bytecode report and upload it as an artifact."
```